assets/node.yaml

```diff
@@ -57,6 +57,8 @@ spec:
             - csi-driver-shared-resource
           args:
             - --config=/var/run/configmaps/config/config.yaml
+            - --config=/var/run/configmaps/config/sharedconfigmap-list.yaml
+            - --config=/var/run/configmaps/config/sharedsecret-list.yaml
```

Comment:
I think that you need to remove these two lines now?

Reply:
I am not sure about this because https://github.com/openshift/csi-driver-shared-resource-operator/blob/master/assets/node.yaml#L59 this line only points to config.yaml file